.env.example

```
@@ -0,0 +1,9 @@
+POSTGRES_USER = postgres
+POSTGRES_PASSWORD = postgres
+POSTGRES_DB = postgres
+POSTGRES_PORT = 5432
+POSTGRES_HOST = localhost
+
+SECRET = secret_key
+ALGORITHM = HS256
+
```

```suggestion
PGADMIN_DEFAULT_EMAIL = 'postgres@postgres.com'
PGADMIN_DEFAULT_PASSWORD = 'postgres'
```
Add pgadmin settings